var/spack/repos/builtin/packages/libgit2/package.py

```diff
@@ -16,7 +16,7 @@ class Libgit2(CMakePackage):
     homepage = "https://libgit2.github.com/"
     url = "https://github.com/libgit2/libgit2/archive/v0.26.0.tar.gz"
 
-    maintainers = ["AndrewGaspar"]
+    maintainers = []
```

Comment:
This works though, should the class ever inherit from a package that has maintainers, this will override that setting.